.github/workflows/queries.yml

```diff
@@ -0,0 +1,44 @@
+name: Cypher Queries
+
+on:
+  push:
+    branches:
+      - 'publish'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    services:
+      neo4j:
+        image: neo4j:4.3
+        ports:
+          - 7687:7687
+        env:
+          NEO4J_AUTH: neo4j/s3cr3t
+          NEO4J_ACCEPT_LICENSE_AGREEMENT: yes
+          NEO4JLABS_PLUGINS: '["apoc", "graph-data-science"]'
+        options: >-
+          --health-cmd "cypher-shell -u neo4j -p s3cr3t 'RETURN 1'"
+          --health-interval 5s
+          --health-timeout 60s
+          --health-retries 12
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 10
+        uses: actions/setup-node@v1
+        with:
+          # remind: Antora runs faster on Node 10
+          node-version: '10'
```

Comment:
Node 10 is not maintained anymore: https://nodejs.org/en/about/releases/

Reply:
We are using Node 10 because Asciidoctor.js (or more precisely Opal) is 3 times slower on Node > 10: https://github.com/asciidoctor/asciidoctor.js/pull/1228

Antora 3.x will use the latest version of Asciidoctor.js where it's now only ~1.5 times slower.

Having said that, the build is relatively fast so we can try to use Node 14 and see how it goes.